.travis.yml

```diff
@@ -18,7 +18,11 @@ env:
   - secure: "IhEskTMEAsBNYTQCpDSlvDxIJK5cu1r2V"
 
 before_install:
-  - sudo apt-get update
+  - wget -qO - https://packages.confluent.io/deb/5.3/archive.key | sudo apt-key add -
```

Comment:
have you also included all the .sh files and Makefile alterations for the CI to run ?

Reply:
thanks for reviewing, and the changes in '.travis.yml' file contain the whole thing to make CI environment work